### PR TITLE
64 mark book as out of stock

### LINKED ISSUE #64: Mark book as out of stock
Create a api where we can mark book as out of stock

### PR DESCRIPTION
**Pull request by manas-chandan**
- [x] define method for out of stock book
- [x] implemented method for out of stock book
- [x] controller method for out of stock book add

### CODE REVIEW

File: backend/src/main/java/com/lms/backend/services/impl/BookServiceImpl.java
Comment (smruti-banty) [inline backend/src/main/java/com/lms/backend/services/impl/BookServiceImpl.java:1]: set updated date

**Review by smruti-banty — CHANGES_REQUESTED** [review]
Change accordingly

**Comment by manas-chandan** [comment]
changes done accordingly.

**Review by smruti-banty — APPROVED** [review]
LGTM.

**Pull request merged**